packages/origin-ui-core/src/components/Organization/PlatformOrganizationRegistrationForm.tsx

```diff
@@ -84,7 +84,13 @@ const VALIDATION_SCHEMA = Yup.object({
     signatoryCountry: Yup.string().required().label('Country')
 });
 
-export function PlatformOrganizationRegistrationForm() {
+interface IProps {
+    nextStep: (step: STEP_NAMES) => void;
+}
+
+export function PlatformOrganizationRegistrationForm(props: IProps) {
+    const { nextStep } = props;
+    /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Trying to understand why this is needed? Don't see any new unused variables.

If it's needed for the whole file, should be put on the top of the file.
If it's only for one line, should be used as `// eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
Its temporarily until new organization registration endpoint is ready